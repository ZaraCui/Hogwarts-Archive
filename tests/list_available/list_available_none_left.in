ADD COLLECTION data/books_small.csv
ADD STUDENT Harry Potter
RENT 100000 1001
RENT 100000 1002
RENT 100000 1003
RENT 100000 1004
LIST AVAILABLE
EXIT

