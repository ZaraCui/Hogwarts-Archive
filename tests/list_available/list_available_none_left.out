user: No such collection.

user: Success.

user: No spellbooks in system.

user: No spellbooks in system.

user: No spellbooks in system.

user: No spellbooks in system.

user: No spellbooks in system.

user: Ending Archive process.
